modules/ROOT/pages/manage-versions-instances-concept.adoc

```diff
@@ -3,25 +3,24 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-After you create an API in Design Center for example, you set a version name and publish it to Exchange. In this release, the version resides in Exchange, not API Manager. You then set up API Manager to manage the version.
+After you create an API in Anypoint Paltform, you configure the version details for the API and then publish it to Exchange. The API version resides in Exchange, and not in API Manager. However, you then manage this API version from API Manager:
 
 image::environment2.png[]
 
-== API Instance Management in API Manager
+== Managing API Instances in API Manager
 
-In API Manager, you may also create, import, and deprecate API instances.
+You can create, import, and deprecate API instances using API Manager. You can use instances of an API in multiple environments. You can also have multiple instances of an API in the same environment:
 
-On the API administration dashboard, you expand the API version name to display the instances if there are any:
+//image::jsonplaceholder2.png[]
 
-image::jsonplaceholder2.png[]
+You can create one instance of an API that serves as a proxy and create another to directly manage as a basic endpoint. You can apply policies to the API based on how you have configured the endpint. For example, you can apply caching policies to the proxy, and throttling policies and security policies to the basic endpoint. 
 
-You can use instances of an API in multiple environments, and you can have multiple instances in the same environment. You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies to the proxy, for example, and throttling policies and security policies to the basic endpoint. For more information about API instance management, see "About Basic API Management Tasks" link below.
+Additionally, you can promote an API to another environment.
```

Comment:
Is this called "promoting"? Does that mean that you can share the API, create a new instance, or move the API entirely? Can you then change the shared/moved instance independently of the original?

Reply:
No promote is OK. And promote means that you can move an API from one environment to another, for example, from testing to production.